include/swift/AST/DiagnosticsParse.def

```diff
@@ -1964,6 +1964,13 @@ ERROR(attr_requires_concurrency, none,
       "concurrency is enabled",
       (StringRef, bool))
 
+//------------------------------------------------------------------------------
+// MARK: Variadics
+//------------------------------------------------------------------------------
+
+ERROR(associatedtype_cannot_be_pack,none,
+      "associated types may not use type parameter packs", ())
```

Comment:
'associated types cannot be variadic' perhaps?

Reply:
Sounds good to me!